_FIPS201/requirements.md

```diff
@@ -485,7 +485,10 @@ management key ([Section 4.2.2.5](frontend.md#s-4-2-2-5)) was compromised, the c
 
 ### 2.9.3 PIV Card Activation Reset {#s-2-9-3}
 
-The Personal Identification Number (PIN) on a PIV Card may need to be reset if the cardholder has
+The activation methods for a PIV Card, including both Personal Identification Number (PIN) and OCC, 
```

Comment:
The new text seems to imply that other card activations methods are possible, since it states ..."including PIN and OCC".  We might need to tighten that up.  How about:

The Person Identification Number (PIN) and OCC activate the PIV Card.  Both OCC and PIN might need to be reset under certain circumstances as described in this section.

Reply:
```suggestion
Both the Personal Identification Number (PIN) and OCC activation methods for a PIV Card
```